dnf-behave-tests/features/epoch.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Test upgrading and downgrading to package with different epoch
+
+
+Scenario: Upgrade to RPM with same NVR but different epoch
+  Given I use repository "epoch0"
```

Comment:
How about naming the repos `epoch-[1-3]`?

Reply:
It's named this way, so that the name of the repository corresponds to the epoch defined in spec. So, repositories ``epoch0`` and ``epoch1`` contain packages with epochs ``0`` and ``1``, respectively, and repository ``epoch`` contains package without explicit epoch definition. I find it easy to work with. Is there any reason for preferring ``epoch-[1-3]`` naming?

Reply:
I see, I missed that the package without an epoch is in a repository with no number in its name.

I was just thinking we should keep the convention of naming the repos according to the feature files and in case there are more, name them e.g. `epoch-something`, `epoch-something-else` etc. But your naming makes sense and I don't have a better idea so we can keep it as it is...